ui/src/features/Editor/components/OverlayUI/components/DebugPanel/DebugPreview/hooks.ts

```diff
@@ -0,0 +1,39 @@
+import bbox from "@turf/bbox";
+import { LngLatBounds } from "maplibre-gl";
+import { useCallback, useMemo } from "react";
+
+export default ({
+  mapRef,
+  selectedOutputData,
+}: {
+  mapRef: any;
+  selectedOutputData: any;
+}) => {
+  const dataBounds = useMemo(() => {
```

Comment:
Since this only seems to be used in handleMapLoad, just move that code into there